tsconfig.cjs.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "module": "commonjs",
+    "outDir": "./dist/cjs",
+    "target": "ES5"
+  },
+  "include": [
+    "dist/cjs/src/**/*"
+  ],
+  // XXX: if we don't specify this, tsc adds the value of `outDir` to `exclude`
```

Comment:
this seems bit odd. I remember it shouldn't be needed with correct config inherits. what brings this for our config?